### PR TITLE
Issue #26 Create InvoiceEntity & enum InvoiceStatus

### LINKED ISSUE #26: Create InvoiceEntity

### PR DESCRIPTION
**Pull request by nikaalin**

### CODE REVIEW

**Review by a-deriabin — APPROVED** [review]

**Review by JustRevolt — APPROVED** [review]

**Review by ChirkinaTanya — APPROVED** [review]

File: src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        this.id = id;
+        this.orderId = orderId;
+        this.issueDate = issueDate;
+        this.paymentDate = paymentDate;
+        this.totalPrice = totalPrice;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java]: Where is braces {} ?
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf 12 page

File: src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+import java.util.Date;
+import java.util.Objects;
+
+public class InvoiceEntity {
+    private int id;
+    private int orderId;
+    private Date issueDate;
+    private Date paymentDate;
+    private int totalPrice;
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java]: Why int?

**Review by PavelEzhkov — COMMENTED** [review]
Fix it

**Review by PavelEzhkov — COMMENTED** [review]
One more thing

File: src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java:16]: Where is invoice status?

**Review by PavelEzhkov — CHANGES_REQUESTED** [review]

**Review by PavelEzhkov — APPROVED** [review]

**Pull request merged**